packages/connect-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-client-certificate.tsx

```diff
@@ -49,7 +49,7 @@ function TLSClientCertificate({
           }}
           showFileOnNewLine
           optional
-          optionalMessage="Optional (only required with X.509)"
+          optionalMessage="Optional (required with X.509)"
```

Comment:
I’d suggest

```suggestion
          optionalMessage="Optional (required with X.509 auth)"
```

or

```suggestion
          optionalMessage="Optional (required with X.509 authentication)"
```

if we have the space for displaying that (since X.509 usage != X.509 authentication)